src/components/plots/ContinuousEmbeddingPlot.jsx

```diff
@@ -26,10 +25,9 @@ const ContinuousEmbeddingPlot = (props) => {
     loading, error,
   } = useSelector((state) => state.embeddings[embeddingSettings.method]) || {};
   const geneExpression = useSelector((state) => state.genes.expression);
-  const geneProperties = useSelector((state) => state.genes.propreties);
   const cellSets = useSelector((state) => state.cellSets);
   const [plotSpec, setPlotSpec] = useState({});
-  const PROPERTIES = ['dispersions'];
+  const { fetching } = useSelector((state) => state.genes.properties.views[plotUuid]) || false;
```

Comment:
this line of code will break if there is nothing under `genes` or under `properties` or under `views`, you should add the `?` after each of those.

Reply:
Shouldn't it be equal to false with the || operator ?

Reply:
have you checked? Better use `?` instead, to keep this code consistent with the rest of the code.